docs/compilers/Dynamic Analysis/MetadataFormat.md

```diff
@@ -0,0 +1,48 @@
+#### <a name="MethodSpansBlob"></a>Method Spans Blob
+Span is a quadruple of integers and a document reference:
+
+* Start Line
+* Start Column
+* End Line
+* End Column
+* Document
+
+The values of a span must satisfy the following constraints
+
+* Start Line is within range [0, 0x20000000).
+* End Line is within range [0, 0x20000000).
+* Start Column is within range [0, 0x10000)
+* End Column is within range [0, 0x10000)
+* End Line is greater or equal to Start Line.
+* If Start Line is equal to End Line then End Column is greater than Start Column.
+
+_Method Spans Blob_ has the following structure:
+
+    Blob ::= header span-record (span-record | document-record)*
+
+#####header
+| component        | value stored            | integer representation |
+|:-----------------|:------------------------|:-----------------------|
+| _InitialDocument_| Document row id         | unsigned compressed    |
+
+#####sequence-point-record
+| component      | value stored                                                 | integer representation                      |
+|:---------------|:-------------------------------------------------------------|:--------------------------------------------|
+| _ΔLines_       | _EndLine_ - _StartLine_                                      | unsigned compressed                         |
+| _ΔColumns_     | _EndColumn_ - _StartColumn_                                  | _ΔLines_ = 0: unsigned compressed, non-zero |
+|                |                                                              | _ΔLines_ > 0: signed compressed             |
+| _δStartLine_   | _StartLine_ if this is the first non-hidden sequence point   | unsigned compressed                         |
```

Comment:
For our initial venture into code, the spans that occur match up pretty well with sequence points. After we add other sorts of dynamic analysis, e.g. stuff about calls or other pieces of statements, the spans become arbitrary regions of text--essentially anything that can be encoded in the location of the syntax node of a bound node. Taking the more general approach happens to work fine for dealing with the requirements of simple code coverage without introducing complexity, so we might as well have full generality from the start.